css-pseudo-classes/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>CSS Pseudo Classes</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <p class="title">Hover Button</p>
```

Comment:
Rather than a `<p>` tag, I would've made this an `<h2>`. `h2`'s are already bold by default, and already have a bigger `font-size` because they are headings.